Src/Particle/AMReX_NeighborParticlesGPUImpl.H

```diff
@@ -255,6 +255,10 @@ updateNeighborsGPU (bool boundary_neighbors_only)
     }
 
     clearNeighbors();
+
+    snd_buffer.setArena(The_Comms_Arena());
```

Comment:
Should these two lines inside `if (ParallelDescriptor::UseGpuAwareMpi())` like in AMReX_ParticleContainerI.H?

Reply:
Yes.